### ISSUE TITLE
bpftrace: compilation error.

### ISSUE THREAD

**Issue by rgrunbla** [NixOS/nixpkgs#105228]
**Describe the bug**
Trying to compile bpftrace on NixOS, the compilation fails. The log is provided as an attachment.

**To Reproduce**
1. `# nix-channel --update nixpkgs`
2. `# nix-env -iA nixos.linuxPackages.bpftrace`

**Expected behavior**

Being able to install bpftrace.

**Additional context**

The current version of bpftrace is also slightly out-of-date (v11.0 vs v11.4).

**Notify maintainers**

@thoughtpolice @rvl

**Metadata**

```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.9.8, NixOS, 21.03pre251971.a371c107116 (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.8`
 - channels(remy): `""`
 - channels(root): `"nixpkgs-21.03pre254169.4f165c6d7d4, nixos-21.03pre253635.2247d824fe0"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

[log.txt](https://github.com/NixOS/nixpkgs/files/5610591/log.txt)

**Comment by rvl** [comment]
Thanks for the report. This is fixed by #105229 (nixos-unstable) and #105415 (nixos-20.09).

**Comment by rgrunbla** [comment]
Thanks !
